xcode/Subconscious/Shared/Components/FirstRun/FirstRunView.swift

```diff
@@ -77,24 +113,27 @@ struct FirstRunView: View {
                         }
                     )
                     .buttonStyle(PillButtonStyle())
-                    .disabled(!app.state.inviteCodeFormField.isValid)
+                    .disabled(app.state.gatewayId == nil)
```

Comment:
You cannot proceed without a gateway ID, unless going via the "Use Offline" path.